mass_mailing_partner/tests/test_partner_mail_list_wizard.py

```diff
@@ -24,6 +25,7 @@ def test_add_to_mail_list(self):
         # This line does not create a new contact
         wizard.add_to_mail_list()
         self.assertEqual(len(self.partner.mass_mailing_contact_ids), 1)
+
```

Comment:
Why this extra empty line? PEP8 discourages to introduce these lines inside a method.

Reply:
Deleted!